eng/spotbugs-aggregate-report/pom.xml

```diff
@@ -64,18 +64,17 @@
                 <source>..\..\sdk\eventhubs\azure-eventhubs\src\main\java</source>
                 <source>..\..\sdk\eventhubs\azure-eventhubs\src\samples\java</source>
                 <source>..\..\sdk\identity\azure-identity\src\main\java</source>
-                <source>..\..\sdk\identity\azure-identity\src\samples\java</source>
```

Comment:
Why you remove this line? 

And why we common out the rest?

Reply:
Merge anomaly, added the line back.